src/meshapi/admin/urls.py

```diff
@@ -1,17 +1,19 @@
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, re_path
 
 from meshapi.admin.password_reset import (
     AdminPasswordResetCompleteView,
     AdminPasswordResetConfirmView,
     AdminPasswordResetDoneView,
     AdminPasswordResetView,
 )
+from meshdb.views import admin_iframe_view
 
 urlpatterns = [
     path("password_reset/", AdminPasswordResetView.as_view(), name="admin_password_reset"),
     path("password_reset/done/", AdminPasswordResetDoneView.as_view(), name="password_reset_done"),
     path("password_reset/<uidb64>/<token>/", AdminPasswordResetConfirmView.as_view(), name="password_reset_confirm"),
     path("password_reset/done/", AdminPasswordResetCompleteView.as_view(), name="password_reset_complete"),
+    re_path(r"^panel/.*$", admin_iframe_view),  # Match any other /admin/* URL
```

Comment:
Does this still need to be so broad? With the JS solution, why does it still need a regex? Also perhaps we should be a bit more descriptive and call it `"iframe_wrapper/"` or something

Reply:
Evidently not, actually.